packages/cli/__tests__/provisioning/__system__/perform/cli.provisioning.perform.action.system.test.ts

```diff
@@ -13,7 +13,7 @@ import { Imperative, Session } from "@zowe/imperative";
 import { runCliScript } from "../../../../../../__tests__/__src__/TestUtils";
 import { ITestEnvironment } from "../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
 import { TestEnvironment } from "../../../../../../__tests__/__src__/environment/TestEnvironment";
-import { ProvisioningConstants } from "../../../../../../packages/provisioning";
+import { ProvisioningConstants } from "@zowe/provisioning-for-zowe-sdk";
 import * as fs from "fs";
 import { ProvisioningTestUtils } from "../../../../../../packages/provisioning/__tests__/__resources__/utils/ProvisioningTestUtils";
```

Comment:
I'm hoping that in the future we can do [relative imports](https://www.typescriptlang.org/docs/handbook/module-resolution.html#relative-vs-non-relative-module-imports)